### PR TITLE
Fix: Eslint issues any types and others

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Fix last eslint issues.

# Changes

* Fix no-any lint issue.
* Add comments enabling eslint issues.

# Tests

No new tests.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis @peterpeterparker please review

Last eslint issues! I also added `npm run lint` to `npm run check`. Therefore, the pipeline will be red when new changes don't follow eslint rules.

File: frontend/svelte/src/lib/utils/utils.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
+/* eslint-disable-next-line @typescript-eslint/ban-types */
 export const debounce = (func: Function, timeout?: number) => {
   let timer: NodeJS.Timer | undefined;
 
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
   return (...args: any[]) => {
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/utils/utils.ts:6]: will it work with `: unknown[]`?

**Review by mstrasinskis — APPROVED** [review]
LGTM! Just one question out of curiosity :)

File: frontend/svelte/src/lib/utils/utils.ts (hunk 1 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/utils/utils.ts]: Yes, like a charm!

**Pull request merged**